lua/bropilot/util.lua

```diff
@@ -64,6 +64,18 @@ local function finish_progress(handle)
   end
 end
 
+---@param tbl string[]
+---@param x string
+local function contains(tbl, x)
+  local found = false
+  for _, v in pairs(tbl) do
+    if v == x then
+      found = true
+    end
+  end
+  return found
```

Comment:
and if we end up returning true inside the loop, we can just ignore the `found` variable and return `false` here